store/postgres/src/deployment_store.rs

```diff
@@ -702,6 +702,17 @@ impl DeploymentStore {
             let schema_name = site.namespace.clone();
             let layout = store.layout(conn, site)?;
             let table = resolve_table_name(&layout, &entity_name)?;
+
+            // If no index method is specified, we default to BTree if the
+            // table is not immutable, and GIST otherwise
+            let index_method = index_method.unwrap_or_else(|| {
+                if table.immutable {
+                    Method::BTree
+                } else {
+                    Method::Gist
+                }
+            });
```

Comment:
Wouldn't that also create a GiST index for an index on two numbers? We should only create a GiST index when one of the indexed columns is a range type (i.e., `block_range`)

Reply:
Should i be reusing this method? Or should i just check if any of the fields is `block_range` and just set Gist if thats the case?

https://github.com/graphprotocol/graph-node/blob/cea3ad277ffd481f0c6b27fa77390876d79345c3/store/postgres/src/relational/ddl.rs#L202-L250